includes/class-wc-payments-customer-service.php

```diff
@@ -249,4 +284,13 @@ private function recreate_customer( $user, $name, $email ) {
 
 		return $this->create_customer_for_user( $user, $name, $email );
 	}
+
+	/**
+	 * Returns the name of the customer option meta, taking test mode into account.
+	 */
+	private function get_customer_id_option() {
```

Comment:
We could use a return type here.

Reply:
Good catch! Fixed in https://github.com/Automattic/woocommerce-payments/pull/972/commits/1c13286f90528e783d69c14fe648ef36742e1024.

Reply:
Looks good. I'd also suggest going one step forward to `private function get_customer_id_option(): array `

Reply:
Gotcha. Added in https://github.com/Automattic/woocommerce-payments/pull/972/commits/eb35c0eb78232d83b751d905cfb399b42aab38b2.

Reply:
Love it, thanks!